commands/droplet_actions.go

```diff
@@ -544,3 +548,15 @@ func RunDropletActionSnapshot(c *CmdConfig) error {
 
 	return performAction(c, fn)
 }
+
+// contextualAtoi cleans the error output of Atoi calls
+func contextualAtoi(s, resource string) (int, error) {
+	n, err := strconv.Atoi(s)
+	if err == nil {
+		return n, nil
```

Comment:
You should check if `n` is positive before returning here since the returned error on line 559 indicates a positive integer is expected.

Reply:
I think you can ignore my suggestion as that's not really the scope of the issue reported, and it would make sense to implement that across the entire cli, not just for droplet actions.

Other than @andrewsomething request bellow, I think this looks good.